environments/conda/environment.yml

```diff
@@ -6,23 +6,25 @@ channels:
     - conda-forge
```

Comment:
can we add the pytorch and nvidia channels above conda-forge here working toward replicating

```sh
conda install pytorch torchvision pytorch-cuda=12.1 -c pytorch -c nvidia
```

which is used in the official [torch Dockerfile](https://github.com/pytorch/pytorch/blob/a0cea517e770fa04f1f33bf6847c6852ca9abc06/Dockerfile#L77)